mods/screwdriver/init.lua

```diff
@@ -80,6 +130,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 	if should_rotate then
 		node.param2 = new_param2
 		minetest.swap_node(pos, node)
+		minetest.check_for_falling(pos)
```

Comment:
This catches things where people try to exploit broken mods that don't call nodeupdate and try to create floating nodes. The nodeupdate call will force nodes around the rotated item to fall and/or break. So it's never a bad idea.

Also, if there are no updates, there are no packets, so it's really cheap.